src/coreclr/interpreter/interpretershared.h

```diff
@@ -28,6 +30,7 @@ struct InterpMethod
     CORINFO_METHOD_HANDLE methodHnd;
     int32_t allocaSize;
     void** pDataItems;
+    CallStubHeader *pCallStub;
```

Comment:
This should have a comment indicating its for use for calling from the jitted calling convention into the interpreter